requirements-tests.txt

```diff
@@ -10,3 +10,19 @@ coverage==3.7
 requests==2.0.1
 six==1.4.1
 waitress==0.8.7
+boto3==1.4.0
+boto==2.42.0
```

Comment:
requirements-tests.txt should contain the whole dependency tree as this is a state that is known to work. I don't want failed tests if boto or any other sub-dependency updates with incompatible changes.